packages/tracing/src/idletransaction.ts

```diff
@@ -119,7 +119,7 @@ export class IdleTransaction extends Transaction {
 
     if (this._heartbeatCounter >= 3) {
       logger.log(
-        `[Tracing] Transaction: ${SpanStatus.Cancelled} -> Heartbeat safeguard kicked in since content hasn't changed for 3 beats`,
+        `[Tracing] Transaction: ${SpanStatus.DeadlineExceeded} -> Heartbeat safeguard kicked in since content hasn't changed for 3 beats`,
```

Comment:
I guess the status doesn't need to be part of the message?
Shorter messages == smaller bundle size.

```suggestion
        `[Tracing] Transaction finished because of no change for 3 heart beats`,
```

Reply:
Good point, will change.